mbbsd/mail.c

```diff
@@ -1194,6 +1194,101 @@ mail_all(void)
     return 0;
 }
 
+void
+get_account_manager(struct Vector *namelist)
+{
+    FILE *fp;
+    if ( (fp = fopen(FN_MAIL_ACCOUNT_SYSOP, "r")) != NULL) {
+        char line[STRLEN+1];
+        while(fgets(line, STRLEN + 1, fp)) {
+            char *ptr = strstr(line, "\n");
```

Comment:
fgets guarantee '\0' in the buffer. So these 4 lines could be replaced by
chomp(line);

https://github.com/ptt/pttbbs/blob/master/common/sys/string.c#L67